cli/src/commands/bookmark/create.rs

```diff
@@ -81,9 +81,30 @@ pub fn cmd_bookmark_create(
     }
 
     let mut tx = workspace_command.start_transaction();
+    let sync_settings = tx.settings().sync_settings()?;
+    let readonly_repo = tx.base_repo().clone();
     for name in bookmark_names {
         tx.repo_mut()
             .set_local_bookmark_target(name, RefTarget::normal(target_commit.id().clone()));
+        for pattern in sync_settings
+            .auto_track_bookmarks
+            .iter()
+            .filter(|pattern| pattern.bookmark.is_match(name.as_str()))
+        {
+            for (remote, view) in readonly_repo
+                .view()
+                .remote_views_matching(&pattern.remote.to_matcher())
+            {
+                let symbol = name.to_remote_symbol(remote);
+                if view.bookmarks.contains_key(name) {
+                    writeln!(
+                        ui.warning_default(),
+                        "Auto-tracking bookmark that exists on the remote: {symbol}"
+                    )?;
+                }
```

Comment:
Can you add a test for patterns with multiple matches (e.g. `["glob:*@*", "glob:*@*"]`)? `track_remote_bookmarks()` should be noop if the bookmark has been tracked, but a warning shouldn't be emitted in that case.

Another idea is to extend the check for `has_tracked_remote_bookmarks()`. If there are untracked remote bookmarks that match the auto-track patterns, the local bookmark name could be considered "existing".

Reply:
Now that the pattern is under a `[remotes.<name>]` section, it's not possible anymore for a bookmark to match multiple patterns per remote.